ReleaseNotes.txt

```diff
@@ -1,4 +1,30 @@
 4.0.16
+New function Get-BcEnvironmentsUsedStorage, to retrieve a list of used storare.
```

Comment:
Nice.
I had to ship 4.0.16 yesterday due to an urgent bug, these things will be in 4.0.17 - you probably have a conflict to resolve.